src/utils/treasure_and_victory_cards.js

```diff
@@ -0,0 +1,17 @@
+const BASE_TREASURE_CARDS = {
+  Copper: { cost: 0, value: 1, type: "TREASURE" },
+  Silver: { cost: 3, value: 2, type: "TREASURE" },
+  Gold: { cost: 6, value: 3, type: "TREASURE" }
+};
+
+const BASE_VICTORY_CARDS = {
```

Comment:
this can be used [here](https://github.com/mdcurrie/dominion/blob/master/src/reducers/game/score.js#L32)